grails-app/views/order/_summary.gsp

```diff
@@ -70,12 +70,21 @@
         </g:if>
 
         <g:if test="${orderInstance?.id}">
-            <g:isSuperuser>
-                <g:link controller="order" action="rollbackOrderStatus" id="${orderInstance?.id}" class="button">
-                    <img src="${resource(dir: 'images/icons/silk', file: 'arrow_undo.png')}" />&nbsp;
-                    ${warehouse.message(code: 'default.button.rollback.label')}
-                </g:link>
-            </g:isSuperuser>
+            <g:hasRoleApprover>
```

Comment:
Since we have hasRoleApprover, could you change this line:
https://github.com/openboxes/openboxes/blob/develop/grails-app/views/purchaseOrderWorkflow/purchaseOrder/showOrderItems.gsp#L16 too?